### DISCUSSION TITLE
New feature request:In a human-in-the-loop setup, besides having 'approved' and 'rejected' buttons, can we add a third option like 'edit' for manually adjusting tool args?

### DISCUSSION THREAD

**Discussion by Onelevenvy** [StreetLamb/tribe#89] [Ideas]
It's an amazine open-source project, thank you!
In a human-in-the-loop setup, besides having 'approved' and 'rejected' buttons, can we add a third option like 'edit' for manually adjusting tool settings? So we can manually update the state when the parameters of the tools are unreasonabl

**Comment by StreetLamb** [comment]
@Onelevenvy Thanks for the suggestion! I am actually exploring allowing users to submit their rejection reason so that agents can re-think the tool parameters or even the choice of tools. Will that be helpful?

**Reply by Onelevenvy** [reply]
Thanks for your  reply! That would actually be quite helpful for me。

**Reply by StreetLamb** [reply]
Hi @Onelevenvy , I have added this feature in v0.6.4. Please see the updated [tutorial](https://github.com/StreetLamb/tribe?tab=readme-ov-file#requiring-human-approval-before-skill-execution-in-sequential-workflows) for how it works. Do let me know if there are any issues!

**Reply by Onelevenvy** [reply]
Hi! That's great news! I've upgraded to v0.6.4, and everything seems to be working fine so far. Thanks for letting me know!
